esengine/bases/result.py

```diff
@@ -4,17 +4,18 @@
 import elasticsearch.helpers as eh
 from six import text_type
 
+HITS = 'hits'
```

Comment:
The string 'hits' is used in many places. If this value change, it is need to change the value in every place. 
So it's easier to maintain and understand.